### PR TITLE
Feature/initial release

### PR DESCRIPTION
**Pull request by atxplangl**

### CODE REVIEW

File: shared-keycloak-app.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
+module "shared_keycloak" {
```
Comment (jjno91) [inline shared-keycloak-app.tf]: this naming feels weird to me, it implies that there will be other keycloak configs in this module

this module should be representing the single keycloak module that will ever exist for the company

File: aurora-keycloak.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module "aurora_keycloak" {
```
Comment (jjno91) [inline aurora-keycloak.tf]: naming here implies that there will be aurora clusters other than the keycloak aurora cluster in this config

**Review by jjno91 — APPROVED** [review]

File: aurora-keycloak.tf [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  source                    = "github.com/champ-oss/terraform-aws-aurora.git?ref=v1.0.19-099cab0"
+  cluster_identifier_prefix = var.git
+  iam_auth_lambda_enabled   = true
```
Comment (reeves122) [inline aurora-keycloak.tf]: Does this need to be enabled all the time for keycloak?

File: shared-keycloak-app.tf (hunk 1 above)
Comment (reeves122) [reply shared-keycloak-app.tf]: Agree, I dont think the "shared" thing is needed

File: shared-keycloak-app.tf [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  # public service vs private service
+  listener_arn = module.core.lb_public_listener_arn
+  lb_dns_name  = module.core.lb_public_dns_name
+  lb_zone_id   = module.core.lb_public_zone_id
+
+  # app specific variables
+  name     = "shared-keycloak"
+  dns_name = local.keycloak_dns_name
+  image    = var.image_shared_keycloak
+  cpu      = "2048"
```
Comment (reeves122) [inline shared-keycloak-app.tf]: We can do this later but I imagine these may need to be variables

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+variable "cloudwatch_slack_url" {
+  description = "channel to post error alert"
+  type        = string
+  default     = ""
+}
+
+variable "db_vendor" {
+  description = "db_vendor"
+  type        = string
+  default     = "mysql"
```
Comment (reeves122) [inline variables.tf:91]: Will keycloak work with anything other than mysql? in any case our Aurora module only supports mysql currently

**Review by reeves122 — DISMISSED** [review]

File: variables.tf (hunk 5 above)
Comment (plangley-champ) [reply variables.tf:91]: it works with Amazon Aurora PostgreSQL as well.  added note in readme and description with mysql w/ aurora is only supported at this time

File: shared-keycloak-app.tf (hunk 1 above)
Comment (plangley-champ) [reply shared-keycloak-app.tf]: updated to keycloak

File: aurora-keycloak.tf (hunk 2 above)
Comment (plangley-champ) [reply aurora-keycloak.tf]: updated to aurora

File: aurora-keycloak.tf (hunk 3 above)
Comment (plangley-champ) [reply aurora-keycloak.tf]: for this module I'll say it is.  If we want to add support for other db including PostgreSQL then I'll create enhancement

**Pull request merged**